# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: security/manager/ssl/tests/unit/test_trust_anchors/ee-notBefore-2023.pem
# Commit: 55e22304ccae
# Full Hash: 55e22304ccaea2af33219077000171821441822c
# Author: Dana Keeler <dkeeler@mozilla.com>
# Date: 2025-08-07 04:23:39
# Description:
#   Bug 1981271 - reset state specific to the built candidate chain root when continuing path building r=nkulatova
#   
#   Any state that NSSCertDBTrustDomain keeps with regard to a built candidate
#   chain (and in particular, the root of that chain), must be reset if path
#   building continues with other potential chains. In particular, this means that
# ==============================================================================

diff -r 2c224e8a6135 -r 55e22304ccae security/manager/ssl/tests/unit/test_trust_anchors/ee-notBefore-2023.pem
--- a/security/manager/ssl/tests/unit/test_trust_anchors/ee-notBefore-2023.pem	Wed Aug 06 18:09:45 2025 +0000
+++ b/security/manager/ssl/tests/unit/test_trust_anchors/ee-notBefore-2023.pem	Wed Aug 06 18:13:58 2025 +0000
@@ -1,5 +1,5 @@
 -----BEGIN CERTIFICATE-----
-MIICxDCCAaygAwIBAgIUQbAFBAJcR+nWt1dATlPDuABJgEAwDQYJKoZIhvcNAQEL
+MIIFvDCCBKSgAwIBAgIURY2ZshgFrEoHGXuloye3tOv4MZkwDQYJKoZIhvcNAQEL
 BQAwDjEMMAoGA1UEAwwDaW50MCIYDzIwMjMwMTAxMDAwMDAwWhgPMjA1MTAxMDEw
 MDAwMDBaMA0xCzAJBgNVBAMMAmVlMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIB
 CgKCAQEAuohRqESOFtZB/W62iAY2ED08E9nq5DVKtOz1aFdsJHvBxyWo4NgfvbGc
@@ -7,11 +7,27 @@
 a2wO7RWCD/F+rWkasdMCOosqQe6ncOAPDY39ZgsrsCSSpH25iGF5kLFXkD3SO8Xg
 uEgfqDfTiEPvJxbYVbdmWqp+ApAvOnsQgAYkzBxsl62WYVu34pYSwHUxowyR3bTK
 9/ytHSXTCe+5Fw6naOGzey8ib2njtIqVYR3uJtYlnauRCE42yxwkBCy/Fosv5fGP
-mRcxuLP+SSP6clHEMdUDrNoYCjXtjQIDAQABoxcwFTATBgNVHSUEDDAKBggrBgEF
-BQcDATANBgkqhkiG9w0BAQsFAAOCAQEAfd1W1LYT+JnTb7ZXdz7lJcwdowimjUWR
-ylhXpqyMbJmldogIoWXG+wPo9XosdLeaR0H7xizrhpiod6DXvqtXUjzfhdzbH8i8
-3sBL3dyO/RAm1IWuDTNmT9d2SX+fty7M7mHH1TLuRda4VItiWyPK+QQIZHcTlhQz
-qRebW6ggpWzRb9nqUWieHlvyaVgqWkv9LiCkJYqXXL6nBvQAh8ukf6g127c0hbMO
-DIQtoAT6XFbApM6GPuovaiMf0h8n7S2ekIcRBEeadvZOMsy7hdTNMKlS706wQETd
-U0jwYTk728Oz0MCdgn488iRWGeDJWi544JZldErK75lWHAU5svaHIQ==
+mRcxuLP+SSP6clHEMdUDrNoYCjXtjQIDAQABo4IDDTCCAwkwEwYDVR0lBAwwCgYI
+KwYBBQUHAwEwggLwBgorBgEEAdZ5AgQCBIIC4ASCAtwC2gEvAFQiJZjzPTZIBULa
+7ODmuU3hXA7ujFkUykjXXjxIToA/AAABUfp73AAAAAQBAQCj5erpkpg4AlKjuf2N
+rG7fFEG3F+IoFLXqeqfCyQEd4qhCzQK1itcqYGgJ2+RX8IxeMugUj03tv9CQHTCg
+RZdB6yKmUy+0Rq77GIONVmNS2L7lbT5dEsPfo8qQ+wNINTlgE7Wbx8xpaXKeL1Rc
+ZwTlCrhZUjKAXMTn9Y6zgNbkICnl5b3dzh1ld5lUrACB27pGYpfECI7MRXd9vwwu
+AlUqmeKJ2wW/wVOFcb6sgtc23tUIBMNpGrJDkqg1E/XmI9JLWtSCiGFvbHaKmS9T
+8BT0WuczPzAVgIzGJd208C6lON+XXRkJ2ScwgzmLmOqLBXFPCwl6++IKS5MQ+mGy
+NTTrAHYAKrgwRDO5FN7S8x5CB/JRwXo3oJJoUtkIAgb4Xlc5FioAAAFR+nvcAAAA
+BAMARzBFAiBcdVGfExFQzV2K3iCjvAYwkf+yc3VfMWTs/ctCgApw5gIhAJ600SaE
+9/aku/rWNaN6j9xN6xupfT8Gk67k93MANNRoAS8AMQj2tt1yGAfwFpWETYUCVrZx
+k2CD2705NKBQUlAaKJIAAAGUHyl8AAAABAEBAGcd3EBcpDOTT1ps3OI18QTf3SkU
+9dOxx3bYntvo6mvvErDDaAVj5FqTQt+KXzsAa6Ji4degM4cBfrVCq9wxof2c29Bi
+Q3G5dwvZSunyOD1CAqtMlTeNHV4MR9rKvP7RNvSqCKOeptrzOylsFpIVYOF2QSFs
+BpehOK0c3S7inFcZo4Tsi8qQ5cDFGg7jSSs/d7R+TRBiTjPP34AvWCIpCYgPCpGg
+OdwLBFQqFY2gKehHvYivN2dk1CBSrI5NwUiJ6oPQLl/BzctyGY/TP009qFdoF7Bg
+a+sJn1CSg23XBlB8oe/XBowcVGyIgOFnm70tP/sGG4gIWxaZ0Lf4miKfQp4wDQYJ
+KoZIhvcNAQELBQADggEBAC4UG3bAEcFlvlf3SPTb+tiz83L1PIwv0Pyyu3Q4jhGc
+zt8KWiRcsCcGVUOQg3BLCk5XKFAzJwSoxBPOnrf3GrciHyAg9caJl0yVexy+VmI/
+BHzj2Rx1H4mhAi5OBMGKukt9I7mvK1vxRdOHH2MzCzT66F9ljH394RNyx/ofpDIV
+Iyz0eVJq7zhZ0l4+u6T+gt1ykYXKADrpc0go1QBc3i3jRDGBdBhU7TjddZNuT+Z5
+wpzvT+udXFpmx4gE2vvAEeKZSclviKoblG/s+QXgCFUkfkTJS5kZxIAy0Qt3pQmB
+JjxBC98RBo6FTAH5FGn7rHRHMqN+ZngR39ezOPJIBKM=
 -----END CERTIFICATE-----